### PR TITLE
Move k8s upgrade and e2e other feature tests to private cloud

### PR DESCRIPTION
**Pull request by Sunnatillo**
This PR moves k8s upgrade and e2e other features(not including remediation and pivoting) tests to private cloud.

tested here:
k8s upgrade - https://jenkins.nordix.org/job/metal3-e2e-1-29-1-30-upgrade-test-release-1-7/8/
features https://jenkins.nordix.org/job/metal3-centos-e2e-feature-test-release-1-7-features/11

### CODE REVIEW

**Comment by tuminoid** [comment]
Do we have enough space there?

**Comment by Sunnatillo** [comment]
> Do we have enough space there?

I think yes. We are moving only k8s upgrade and other features(not pivoting and remediation) to private cloud. 
This is also experimental case. if we reach the quota, tests will be delayed not failed.

File: jenkins/jobs/capm3-e2e-tests.pipeline [hunk 1]
```diff
@@ -21,9 +21,12 @@ script {
   }
     echo "Checkout ${ci_git_url} branch ${ci_git_branch}"
 
-  if  ("${GINKGO_FOCUS}" == 'integration' || "${GINKGO_FOCUS}" == 'basic') {
+  if  ("${GINKGO_FOCUS}" == 'integration' || "${GINKGO_FOCUS}" == 'basic')  {
```
Comment (tuminoid) [inline jenkins/jobs/capm3-e2e-tests.pipeline:24]:
```suggestion
  if ( "${GINKGO_FOCUS}" == 'integration' || "${GINKGO_FOCUS}" == 'basic' ) {
```

File: jenkins/jobs/capm3-e2e-tests.pipeline [hunk 2]
```diff
@@ -21,9 +21,12 @@ script {
...
     agent_label = "metal3ci-8c16gb-${IMAGE_OS}"
     TIMEOUT=10800 // 3h
+  } else if  ("${GINKGO_FOCUS}" == 'k8s-upgrade') {
```
Comment (tuminoid) [inline jenkins/jobs/capm3-e2e-tests.pipeline:27]:
```suggestion
  } else if ( "${GINKGO_FOCUS}" == 'k8s-upgrade' ) {
```

**Review by tuminoid — COMMENTED** [review]
Nit: keep the spaces in right places

File: jenkins/jobs/capm3-e2e-tests.pipeline (hunk 2 above)
Comment (Sunnatillo) [reply jenkins/jobs/capm3-e2e-tests.pipeline]: done

File: jenkins/jobs/capm3-e2e-tests.pipeline (hunk 1 above)
Comment (Sunnatillo) [reply jenkins/jobs/capm3-e2e-tests.pipeline]: done

**Comment by Sunnatillo** [comment]
testing here: 
k8s upgrade - https://jenkins.nordix.org/job/metal3-e2e-1-29-1-30-upgrade-test-release-1-7/8/
features https://jenkins.nordix.org/job/metal3-centos-e2e-feature-test-release-1-7-features/11

**Review by tuminoid — COMMENTED** [review]
/approve

**Review by smoshiur1237 — COMMENTED** [review]
/lgtm

**Pull request merged**